scenario/tasks.yaml

```diff
@@ -10,7 +10,9 @@ tasks:
    act_stop_vms: ['StopVms', 'src_cloud']
    act_start_vms: ['StartVms', 'dst_cloud']
    act_start_vms_if_needed: ['StartVmsIfNeeded', 'dst_cloud']
-   act_dissociate_floatingip: ['DissociateFloatingip', 'src_cloud']
+   act_dissociate_floatingip: ['DisassociateAllFloatingips', 'src_cloud']
+   disassociate_floatingip_on_source: ['DisassociateFloatingip', 'src_cloud']
```

Comment:
Can you explain naming of tasks here? One is named act_* but another named without act_